lib/car.rb

```diff
@@ -0,0 +1,36 @@
+class Car
+
+  def initialize(make, monthly_payment, loan_length)
+    @make = make
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+  end
+
+  def make
+    @make = "Ford"
+  end
+
+  def model
+    @model = "Mustang"
+  end
```

Comment:
With the way these methods are written, a Car object would always return "Ford" for the make and "Mustang" for the model. Instead, we want this class to be dynamic so that Car objects can have any make/model.